packages/web/src/prerender.browser.utils.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react'
+
+/* Web side prerender utils, to be used on the browser */
+
+export const isPrerendering = (): boolean => {
+  return !!global?.__REDWOOD__PRERENDERING
+}
+
+export const useIsBrowser = () => {
+  return useMemo(() => {
+    return {
+      browser: !global?.__REDWOOD__PRERENDERING,
+    }
+  }, [])
+}
+
+export const BrowserOnly = ({ children }: { children: React.ReactNode }) => {
```

Comment:
Should we just return a boolean instead of an object here? I was thinking that it could be extended in the future

Reply:
I think it's fair for this to be a boolean. I usually would expect `is` to return a boolean.